pkgs/development/tools/misc/premake/5.nix

```diff
@@ -10,49 +10,58 @@
 
 stdenv.mkDerivation rec {
   pname = "premake5";
-  version = "5.0.0-beta4";
+  version = "5.0.0-beta6";
 
   src = fetchFromGitHub {
     owner = "premake";
     repo = "premake-core";
     rev = "v${version}";
-    sha256 = "sha256-sNLCyIHWDW/8jIrMFCZAqtWsh4SRugqtPR4HaoW/Vzk=";
+    sha256 = "sha256-wMy9uqtlWAB+o/D169CLNQDd80zhsWKyNBXzKxhH7n4=";
   };
 
   buildInputs =
-    [ libuuid ]
+    [
+      libuuid
+    ]
     ++ lib.optionals stdenv.hostPlatform.isDarwin [
       Foundation
       readline
     ];
 
-  patches = [ ./no-curl-ca.patch ];
+  # patches = [ ./no-curl-ca.patch ];
   postPatch =
     ''
       substituteInPlace contrib/curl/premake5.lua \
-        --replace "ca = nil" "ca = '${cacert}/etc/ssl/certs/ca-bundle.crt'"
```

Comment:
if they ever decide to do iteration over known directories like before, this will silently break, no? I'd prefer a patch that would have a chance to fail to apply the next time they touch this code to silent break. What do you think?

Reply:
See previous note. `${cacert}` changes whenever the `cacert` package is rebuilt.

Other than that, I agree with you. This PR started as an effort to fix a Hydra error and I decided to see if we could get a version upgrade in. Not sure it's worth significant extra effort.